Client/Helpers/DynamicFontHelper.swift

```diff
@@ -217,6 +217,7 @@ class DynamicFontHelper: NSObject {
     ///   - maxSize: The maximum size the font can scale - Refer to the human interface guidelines for more information on sizes for each style (optional)
     ///              https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/
     /// - Returns: The UIFont with the specified font size, style and weight
+    @available(*, deprecated, message: "Use preferredFont(withTextStyle:size:weight:) instead")
     func preferredFont(withTextStyle textStyle: UIFont.TextStyle, weight: UIFont.Weight? = nil, maxSize: CGFloat? = nil) -> UIFont {
```

Comment:
I think this increases the number of warnings in the project and make the build fail

Reply:
Ok. Then I'll see if I can get rid of enough deprecation warnings to make the build pass.

Reply:
If so then we could remove that function entirely 😄

Reply:
Not yet. There is still some deprecations left in the code.